### ISSUE TITLE
[AWK] problem Installing dependencies under MacBook-Pro

### ISSUE THREAD

**Issue by noahamato1** [r00t-3xp10it/venom#108]
im getting this error when im trying to install the other dependencies. i have zenity, metasploit, ettercap all installed. 
the program comes up on my screen and im also stuck at the "enter metasploit full path"

![Screen Shot 2021-06-11 at 5 14 18 PM](https://user-images.githubusercontent.com/54610696/121749953-7c79c080-cad9-11eb-8334-7bf93b985143.png)
![Screen Shot 2021-06-11 at 5 15 33 PM](https://user-images.githubusercontent.com/54610696/121749971-7f74b100-cad9-11eb-86d4-c9f3ec81e3d2.png)
![Screen Shot 2021-06-11 at 5 19 00 PM](https://user-images.githubusercontent.com/54610696/121749979-826fa180-cad9-11eb-8b29-f65f8b2c3868.png)

**Comment by r00t-3xp10it** [comment]
Execute this two lines and send me one screenshot ..

```bash
MSFDATA=$(zenity --title="☠ Enter METASPLOIT FULL PATH ☠" --text "example: /usr/share/metasploit-framework" --entry --width 330) > /dev/null 2>&1

if [ -z "$MSFDATA" ]; then echo "ERROR: MSFDATA variable empty ..";else echo "Path: $MSFDATA";fi
```

**Comment by noahamato1** [comment]
yes sorry, i am on a Macbook pro running macOS Big Sur version 11.4. So i executed the two lines you gave me and the same screen that wants me to put the full path came up. i searched in my finders folder for anything related to "metasploit" and nothing came up. 
i downloaded Metasploit from both these websites and none of them seems to be working
https://formulae.brew.sh/cask/metasploit
https://www.metasploit.com/


<img width="920" alt="Screen Shot 2021-06-13 at 9 12 37 AM" src="https://user-images.githubusercontent.com/54610696/121808741-f324d400-cc27-11eb-97be-95d6f2123676.png">

**Comment by r00t-3xp10it** [comment]
[Installing Metasploit Framework on OSX](https://docs.rapid7.com/metasploit/installing-the-metasploit-framework)

**Comment by noahamato1** [comment]
i downloaded it and it still says the same thing. im getting the "\awk: can't open file /etc/issue
 source line number 1" in my terminal window. im going to restore my macbook and start fresh again.

**Comment by r00t-3xp10it** [comment]
Execute this on terminal to see if everything its ok with metasploit..

sudo msfconsole

A Dom, 13/06/2021, 22:48, noahamato1 ***@***.***> escreveu:

**Comment by noahamato1** [comment]
ive looked in my user page /theruiner/  and there isnt a folder named .msf4





![Screen Shot 2021-06-13 at 5 54 08 PM](https://user-images.githubusercontent.com/54610696/121822995-62beb180-cc70-11eb-9e37-dc864d2dd698.png)

**Comment by r00t-3xp10it** [comment]
**HINT: .msf4 <-- its a HIDDEN folder**  

Execute this on terminal to see if **awk** its the problem..
```bash
testme="ip addr: 192.168.1.69"
echo $testme|awk {'print $3'}
```
It should respond: `192.168.1.69`

**Comment by noahamato1** [comment]
thats correct. it did respond with that ip.

**Comment by noahamato1** [comment]
i figured that was a hidden folder. i know windows computer can view them with a setting. but i cant figure out mac os has it

**Comment by r00t-3xp10it** [comment]
`cat file-full-path.log` <--- you can read the ontents of file using **cat** command
 So the error its on BASH `AWK` command under mac os ... you must search on online chats
 how to fix the wak issue under mac os ..
 
 i cant help you with that because i dont have a mac installed on virtual machine now ...
 https://www.ionos.com/digitalguide/server/configuration/showing-hidden-files-on-a-mac/
